src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -123,11 +123,11 @@ private EmailContent constructEmailContent(EmailType emailType,
     protected String[] getDissolutionTeamRecipients(String jurisdiction) {
         switch(jurisdiction) {
             case "scotland":
-                return emailConfig.getEmailRecipientsEdinburgh().split(",");
+                return emailConfig.getEmailRecipientsEdinburgh().replace(" ", "").split(",");
```

Comment:
Would be nice to have a local private 'removeSpaces()' method or similar that takes the list of email addresses and does the replace(" ", "") rather than seeing it repeated here. Minor, so will approve. See what others think.